src/test/resources/insert/class/NEW_Klass.java

```diff
@@ -1,7 +1,11 @@
-class Klass {
-    class InsertedClass {
-        public void main() {
-            System.out.println("Doesn't accept any arguments");
-        }
```

Comment:
I shifted this test case to `nested_class` directory because that made more sense.